dev-libs/granite/granite-5.2.4.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION=0.40
+
+inherit xdg-utils meson vala
+
+DESCRIPTION="Elementary OS library that extends GTK+"
+HOMEPAGE="https://github.com/elementary/granite"
+SRC_URI="https://github.com/elementary/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="doc test"
+
+BDEPEND="
+	$(vala_depend)
+	>=dev-util/meson-0.48.2
+	virtual/pkgconfig
+"
+DEPEND="
+	>=dev-libs/glib-2.50:2
+	dev-libs/libgee:0.8[introspection]
+	>=x11-libs/gtk+-3.22:3[introspection]
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
```

Comment:
Do you need to write this phase in the ebuild at all? I checked and vala.eclass seems to export src_prepare, which should handle this automatically. I'd be happy if you tested it :)

Reply:
If I remove the phase I get the following error:
```
eapply_user (or default) must be called in src_prepare()!
``` 

If I add the stage with just `eapply_user` vala breaks later on due to VALAC not being set, so I think I need it as-is